sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/BoundedWindow.java

```diff
@@ -21,11 +21,10 @@
 import org.joda.time.Instant;
 
 /**
- * A {@code BoundedWindow} represents a finite grouping of elements, with an
- * upper bound (larger timestamps represent more recent data) on the timestamps
- * of elements that can be placed in the window. This finiteness means that for
- * every window, at some point in time, all data for that window will have
- * arrived and can be processed together.
+ * A {@code BoundedWindow} represents window information assigned to data elements.
```

Comment:
I think the subject line for the class should be a bit shorter and then newline and `<p>`